### DISCUSSION TITLE
What does "automatically load uninstall list " mean?

### DISCUSSION THREAD

**Discussion by Dolaaz** [Klocman/Bulk-Crap-Uninstaller#466] [Q&A]
What does "automatically load uninstall list " mean?
![9b8vrw99gbha1](https://user-images.githubusercontent.com/127548801/224417235-a73b1140-10b9-479f-a02d-b4fcde044345.jpg)

**Comment by Klocman** [comment]
Check part 6 in https://htmlpreview.github.io/?https://github.com/Klocman/Bulk-Crap-Uninstaller/blob/master/doc/BCU_manual.html
It loads the mentioned uninstall list from the main BCU directory.
